### PR TITLE
Update TileDB-Java dependency to 0.2.0

### PR DESCRIPTION
**Pull request by gsvic**

### CODE REVIEW

**Comment by Shelnutt2** [comment]
@gsvic can you update the build version to 0.0.3 also?

**Comment by gsvic** [comment]
> @gsvic can you update the build version to 0.0.3 also?

Done

**Comment by gsvic** [comment]
> I forgot, we also need to update the following:
> https://github.com/TileDB-Inc/TileDB-Spark/blob/master/scripts/installTileDBJava.sh
> https://github.com/TileDB-Inc/TileDB-Spark/blob/master/scripts/emr-bootstrap/install-tiledb-spark.sh#L5
> https://github.com/TileDB-Inc/TileDB-Spark/blob/a6a05221af6f4d4d812abc673cbcd358b5a34fc2/scripts/emr-bootstrap/install-tiledb-spark.sh#L16
> 
> Can you start a wiki page on "Updating TileDB-Java version" and add all these steps?

Done!

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ This will create a `build/libs/TileDB-Spark-0.0.2-SNAPSHOT.jar` JAR as well as b
 To load the TileDB Spark Datasource reader, 
 you need to specify the path to built project jar with `--jars` to upload to the Spark cluster.
 
-    $ spark-shell --jars build/libs/TileDB-Spark-0.0.2-SNAPSHOT.jar,/path/to/TileDB-Java-0.1.7-SNAPSHOT.jar
+    $ spark-shell --jars build/libs/TileDB-Spark-0.0.3-SNAPSHOT.jar,/path/to/TileDB-Java-0.1.7-SNAPSHOT.jar
```
Comment (Shelnutt2) [inline README.md]: `Java-0.1.7-SNAPSHOT.jar` -> `Java-0.2.0.jar`

File: scripts/emr-bootstrap/install-tiledb-spark.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 sudo yum install -y cmake3 gcc g++ git &&
 sudo ln -s /usr/bin/cmake3 /usr/bin/cmake &&
-git clone https://github.com/TileDB-Inc/TileDB-Java.git -b 0.1.9 &&
+git clone https://github.com/TileDB-Inc/TileDB-Java.git -b 0.2.0 &&
```
Comment (Shelnutt2) [inline scripts/emr-bootstrap/install-tiledb-spark.sh]: Also below `build/libs/tiledb-spark-metrics-0.0.2.jar` -> `build/libs/tiledb-spark-metrics-0.0.3.jar`

Comment (gsvic) [reply scripts/emr-bootstrap/install-tiledb-spark.sh]: Fixed.

**Pull request merged**